shopfloor_mobile/static/wms/src/components/packaging-qty-picker.js

```diff
@@ -194,14 +195,23 @@ export var PackagingQtyPickerMixin = {
          */
         contained_packaging: function() {
             const self = this;
-            let res = {};
+            let res = {},
+                qty_per_pkg,
+                remaining,
+                elected_next_pkg;
             const packaging = this.sorted_packaging;
             _.forEach(packaging, function(pkg, i) {
-                if (packaging[i + 1]) {
-                    const next_pkg = packaging[i + 1];
+                const next_pkgs = packaging.slice(i + 1);
+                remaining = undefined;
+                _.every(next_pkgs, function(next_pkg) {
```

Comment:
I don't get the usage of `_.every`. It should be used to verify a condition, not too loop on elements and set a var name. No? Am I missing anything?

Reply:
It is also a way to loop and break on a condition

Reply:
you mean an hack? :smile: 
You could achieve the same w/ any other looping func, no?